packages/ui/src/2_molecules/AddressTablePagination/AddressTablePagination.stories.tsx

```diff
@@ -0,0 +1,25 @@
+import { Story } from '@storybook/react';
+
+import React, { ComponentProps } from 'react';
+
+import { noop } from '../../utils';
+import { AddressTablePagination } from './AddressTablePagination';
+
+export default {
+  title: 'Molecule/AddressTablePagination',
+  component: AddressTablePagination,
+};
+
+const Template: Story<ComponentProps<typeof AddressTablePagination>> = args => (
+  <>
+    <AddressTablePagination {...args} />
+  </>
+);
+
+export const Basic = Template.bind({});
+Basic.args = {
+  className: '',
+  dataLayoutId: '',
+  limit: 5,
+  onPageChange: noop,
```

Comment:
I'd add here some `alert` or `console.log` to indicate what's happening (e.g. `console.log` with information about what items we should see).